mw-webapp/src/utils/auth/handleLogIn.ts

```diff
@@ -0,0 +1,14 @@
+import {signInWithRedirect} from "firebase/auth";
+import {auth, provider} from "src/firebase";
+
+export const handleLogIn = async () => {
+  try {
+    await signInWithRedirect(auth, provider);
+  } catch (error) {
+    let errorMessage;
+    if (error instanceof Error) {
+      errorMessage = error.message;
+    }
+    alert(errorMessage);
```

Comment:
move to helper or use simplier method like '??' operator

Reply:
Done